test/e2e/email.integration.test.js

```diff
@@ -86,8 +86,8 @@ describe("email", () => {
         .set("Content-Type", "application/json")
         .set("Accept", "application/json")
         .send({
-          to: "example@opencerts.io",
-          data: ropstenCertificate
+          to: "example@tradetrust.io",
```

Comment:
is this a valid email address?

Reply:
it's `example@`, i'm assuming it's a dummy address and not supposed to use anyway? 😬

Reply:
hahaha ok